azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub.py

```diff
@@ -39,7 +39,12 @@ def _run_op(self, op):
             # as a way of enabling this functionality.  If the ConnectedEvent handler
             # sees this -1, it will send a GetTwinOperation to refresh desired properties.
 
-            if op.feature_name == constant.TWIN_PATCHES:
+            # Ensure_desired_properties is set to True by default, but if changed to false,
```

Comment:
This is apparent from the code - what does this mean though? Should be a comment here explaining why this matters